lib/argument.js

```diff
@@ -97,7 +97,7 @@ class Argument {
    */
 
   choices(values) {
-    this.argChoices = values;
+    this.argChoices = values.slice();
     this.parseArg = (arg, previous) => {
       if (!values.includes(arg)) {
```

Comment:
The uses of `values` in the parse function should be replaced with `this.argChoices` now they may be different.